### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by PRRugebregt**

### CODE REVIEW

File: SwipeBaby/Service/CurrentListManager.swift [hunk 1]
```diff
@@ -70,28 +71,29 @@ class CurrentListManager: CurrentListManageable {
...
+        self.hasCurrentViewController = dependencies.rootViewController
         self.networkFetchable = dependencies.networkManager
         self.partnerStateFetchable = dependencies.partnerState
         self.partnerListManageable = dependencies.partnerListManager
         if let namesAlreadyHad = UserDefaults.standard.object(forKey: "savedList\(gender)") as? [String] {
             self.namesAlreadyHad = namesAlreadyHad
         }
-        /// Binding variables to receive partner updates
-        self.partnerListManageable.partnerBoys.bind { partnerBoys in
+        /// observeing variables to receive partner updates
```
Comment (patturik) [inline SwipeBaby/Service/CurrentListManager.swift:81]: Observeing?

File: SwipeBaby/View Controllers/MatchViewController.swift [hunk 2]
```diff
@@ -75,23 +71,27 @@ class MatchViewController: UIViewController {
...
             noMatchWindow.isHidden = true
             pairButton.isHidden = false
             matchLists()
             matchLabel.text = matchViewModel.partnerName
+        } else {
+            pairButton.isHidden = true
+            tableView.alpha = 0.2
+            noMatchWindow.isHidden = false
+            matchLabel.text = "Match"
         }
```
Comment (patturik) [inline SwipeBaby/View Controllers/MatchViewController.swift:95]: put a space here

**Review by patturik — CHANGES_REQUESTED** [review]
Please change these things before merging

File: SwipeBaby/View Controllers/MatchViewController.swift (hunk 2 above)
Comment (PRRugebregt) [reply SwipeBaby/View Controllers/MatchViewController.swift:96]: Done!

File: SwipeBaby/Service/CurrentListManager.swift (hunk 1 above)
Comment (PRRugebregt) [reply SwipeBaby/Service/CurrentListManager.swift]: Changed to Observing

**Review by patturik — APPROVED** [review]
Looks good

**Pull request merged**